### ISSUE TITLE
Extend Veraison verification API to accept CMWs

### ISSUE THREAD

**Issue by thomas-fossati** [CCC-Attestation/attested-tls-poc#34]
Veraison verification API needs to be extended to accept CMWs.

This task depends on #33

**Comment by yogeshbdeshpande** [comment]
@thomas-fossati I do not think, we need this issue any more based on our discussion today!

**Comment by thomas-fossati** [comment]
Superseded by #37
